_locales/en/messages.json

```diff
@@ -349,6 +349,38 @@
       }
     }
   },
+  "openSearchWebsite": {
+    "message": "Search on userstyles.org",
+    "description": "Text for a link that searches for styles for this current site on userstyles.org"
+  },
+  "searchResultsHeader": {
+    "message": "Results for",
+    "description": "Text for label above search results"
+  },
+  "searchResultRating": {
+    "message": "Rating",
+    "description": "Text for label that shows the search result's rating"
+  },
+  "searchResultInstallCount": {
+    "message": "Installs",
+    "description": "Text for label that shows the number of times a search result was installed"
+  },
+  "searchResultInstall": {
+    "message": "Install",
+    "description": "Text for a button that installs a search result"
+  },
+  "searchResultInstallTooltip": {
+    "message": "Install this search result",
+    "description": "Tooltip for a button that installs a search result"
+  },
+  "searchResultCustomize": {
+    "message": "Customize",
+    "description": "Text for a button that customizes a search result before installing"
+  },
+  "searchResultCustomizeTooltip": {
+    "message": "Customize this style on userstyles.org",
```

Comment:
Worth mentioning that it opens a page: `Open the style page on userstyles.org to customize it` or something